pkg/auth/auth.go

```diff
@@ -163,7 +165,7 @@ type KlothoClaims struct {
 	Email         string `json:"email"`
 	EmailVerified bool   `json:"email_verified"`
 	Name          string `json:"name"`
-	jwt.StandardClaims
+	jwt.RegisteredClaims
```

Comment:
note this change (and the tweak to the `claims.ExpiresAt` line below, which this change required)